### ISSUE TITLE
MacOS Safari dark mode = dark font on dark background

### ISSUE THREAD

**Issue by tymoteuszrogalewski** [tsl0922/ttyd#894]
font color and background color is the same under Safari in dark mode.
After login (bind mode :P) also same issue.
When running midnight commander then...everything is blue - also font same blue as background.

Under Chrome its ok.
version 1.6.3
Please fix this ;-)

**Comment by munkale** [comment]
Same issue for me for Safari on iOS.

Attempted setting client options to:

`theme={“foreground”:”white”,”background”:”black”}`

However no success.

**Comment by tymoteuszrogalewski** [comment]
do not sure if same issue. I have outputed chars on screen - but same color as background.

I did try this:
./ttyd -t 'theme={"foreground": "#ff0000", "background": "#cccccc", "cursor": "#0000ff", "selection": "#eeeeee"}'
but only with half success.
Almost all of the above color definitions works....but except the very one named "foreground" ;-/ ...on Safari. With Chrome all parameters works ok.
